fix:  isspace was not declared in this scope
https://github.com/rofafor/vdr-plugin-satip/commit/c10372bb4f1

Signed-of by: Joerg Bornkessel <hd_brummy@gentoo.org> (03/Sep/2018)
diff -Naur satip-2.2.3.orig/satip.c satip-2.2.3/satip.c
--- satip-2.2.3.orig/satip.c	2018-09-03 13:00:20.000000000 +0200
+++ satip-2.2.3/satip.c	2018-09-03 13:00:54.000000000 +0200
@@ -5,6 +5,7 @@
  *
  */
 
+#include <ctype.h>
 #include <getopt.h>
 #include <vdr/plugin.h>
 #include "common.h"
